lib/classes/logged_browser.rb

```diff
@@ -8,6 +8,6 @@ def call
   end
 
   def browser
-    @browser ||= Watir::Browser.new
+    @browser ||= Watir::Browser.new(:chrome)
```

Comment:
Thanks for PR.   Have you found chrome more stable browser for the driver?  The reason I generally use Firefox is that it seems to get faster releases for new versions.